.github/workflows/test.yml

```diff
@@ -48,3 +53,52 @@ jobs:
                "CodeChecker finds different analyzers when running in " \
                "bazel's sandbox environment!"
           CodeChecker analyzers
+
+  unit_test:
+    name: Unit Test
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set bazel version to 6.5.0
+        run: echo "6.5.0" > .bazelversion
+
+      - name: Setup Bazel
+        uses: bazel-contrib/setup-bazel@0.15.0
+
+      - name: Install python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
```

Comment:
Should we also update recommended prerequisites?

Reply:
This is not the earliest version we support, just didn't want to be [so old](https://github.com/Ericsson/codechecker_bazel/pull/5#discussion_r2150278269). We can submit another patch where we downgrade everything to the earliest possible, if you prefer (it will need many more `update-alternatives`  adjustments though).